EIPS/eip-2997.md

```diff
@@ -0,0 +1,100 @@
+---
+eip: 2997
+title: IMPERSONATECALL Opcode
+author: Sergio Demian Lerner (sergio.d.lerner@gmail.com)
```

Comment:
Please include GitHub handle so you can be reached easily and mentioned by the bot.  Either of the following is fine:
```suggestion
author: Sergio Demian Lerner (@SergioDemianLerner), Sergio Demian Lerner <sergio.d.lerner@gmail.com>
```
```suggestion
author: Sergio Demian Lerner (@SergioDemianLerner)
```